changelog.d/292.bugfix

```diff
@@ -0,0 +1 @@
+Fix a bug introduced in Sygnal 0.7.0 where a dict method is called on an object before it is determined to be a dict, causing TypeErrors.
```

Comment:
I think we usually try to write these from the perspective of the user. Can we describe what triggered this and the previous broken behaviour (a 500 internal server error?)